src/Admin/BaseBlockAdmin.php

```diff
@@ -119,11 +118,6 @@ protected function configurePersistentParameters(): array
         ];
     }
 
-    protected function configureRoutes(RouteCollectionInterface $collection): void
-    {
-        $collection->add('view', $this->getRouterIdParameter().'/view');
```

Comment:
This route was added 8 years ago, without any comment, but it was never implemented.

Reply:
actually 12 years ago, probably on one of the first commits, probably never worked.